visualization/app/codeCharta/ui/heightSettingsPanel/heightSettingsPanel.component.html

```diff
@@ -25,6 +25,16 @@
 	</md-input-container>
 </md-slider-container>
 
+<md-input-container class="md-block">
+	<md-checkbox class="md-primary" ng-model="$ctrl._viewModel.showNodeName" ng-change="$ctrl.applySettingsMetricLabelNodeNameChanged()">
+		Label show node name
+	</md-checkbox>
+	</br>
+	<md-checkbox class="md-primary" ng-model="$ctrl._viewModel.showMetricValue" ng-change="$ctrl.applySettingsMetricLabelValueChanged()">
+		Label show metrics name: value
```

Comment:
Maybe "Label show metric(s)" is enough?

Reply:
What about "Show label node metrics"?
Then the other could be "Show label node names"